### DISCUSSION TITLE
Xiaomi QBKG23LM Documentation incorrect

### DISCUSSION THREAD

**Discussion by WarmCatUK** [Koenkk/zigbee2mqtt#6537] [Devices]
Hi, 
Noticed that the documentation page for the Xiaomi QBKG23LM suggests that the possible actions are as below.
However, the only actions are single and release.
I can go ahead and create a pull request at https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/devices/QBKG23LM.md
if it helps.

Action (enum)
Triggered action (e.g. a button click). Value can be found in the published state on the action property. It's not possible to read (/get) or write (/set) this value. The possible values are: hold, single, double, triple, release.

**Comment by Koenkk** [comment]
Please create a pull request

**Reply by WarmCatUK** [reply]
Done, although checks failed.
https://github.com/Koenkk/zigbee2mqtt.io/pull/656
